rules/linux/process_creation/proc_creation_lnx_password_group_discovery.yml

```diff
@@ -0,0 +1,28 @@
+title: Potential Linux Groups Discovery
+id: 6d506805-389e-4370-8270-da7fd947a0da
+status: experimental
+description: This rule detects adversaries attempting to discover group and permission settings.
+    It looks for process execution related to `groups`, `getent`, or `netstat` commands which could indicate an attempt to gather information on available user accounts and groups, including elevated permissions.
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1069.001/T1069.001.md
+author: Milad Cheraghi
+date: 2024-12-01
+tags:
+    - attack.discovery
+    - attack.t1069
+logsource:
+    product: linux
+    category: process_creation
+detection:
+    selection_getent:
+        Image|endswith: '/getent'
+        CommandLine|contains: 'group'
+    selection_netstat:
+        Image|endswith: '/netstat'
+        CommandLine|contains: 'localgroup'
+    selection_groups:
+        Image|endswith: '/groups'
+    condition: 1 of selection_*
```

Comment:
There is already a rule for group discovery. 676381a6-15ca-4d73-a9c8-6a22e970b90d

Use that instead of creating a new one and also move it to hunting as these typical commands and should be a level low